### ISSUE TITLE
A temporary solution

### ISSUE THREAD

**Issue by Ginxx** [Tencent/wcdb#290]
### The language of WCDB

> Swift

### The version of WCDB

> v1.0.6

### The platform of WCDB

> iOS

### The installation of WCDB

> Cocoapods

### What's the issue?

> Post the outputs or screenshots for errors.
> 
> Explain what you want by example or code **in English**.

``` swift
extension Array where Element: CollectionColumnCodable {}

extension Array where Element: CollectionColumnCodable {}

extension Set where Element: CollectionColumnCodable {}
```

The code changes to this, compiled succeeds, will there be any problem?

**Comment by RingoD** [comment]
No. I already try it at https://github.com/Tencent/wcdb/commit/639e4618ae1e80a3d79b91fa36eb0c8a327acde2 . The tests are not passed.
1. Crash with `string.enumerateLinguisticTags` still exists.
2. Swift fail to recognize the `ColumnCodable` elements in `Array`.
